changelog/@unreleased/pr-753.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: break
+break:
```

Comment:
my only other thought about calling this a `break` and cutting another major rev is would we want to also delete any other deprecated things (like `OkhttpTraceInterceptor`) perhaps??

Reply:
Major revs are cheap, we can dev a few times. If there are other breaks you’d like to make in the same major rev as this one we can merge them around the same time.